### ISSUE TITLE
Chapter 8.2 convnet model (dogs vs cats) test accuracy can not achieve 83.5% as described in the book

### ISSUE THREAD

**Issue by shenchenbing** [fchollet/deep-learning-with-python-notebooks#235]
<Deep Learning with Python 2nd Edition> Chapter 8.2 using a convnet model with data augmentation and dropout to classify images of dogs and cats. The book says that the test accuracy is 83.5% but I tried many times, using same model/code and same inputs but got all test accuracy results below 80%. Can anyone try those codes and get results around 83.5%? Or is the book description not correct?

**Comment by WalkCharlie** [comment]
callbacks = [
    keras.callbacks.ModelCheckpoint(
        filepath="convnet_from_scratch.keras",
        save_best_only=True,
        monitor="val_loss")
]
history = model.fit(
    train_dataset,
    epochs=30,
    validation_data=validation_dataset,
    callbacks=callbacks)这步模型训练报错，input arguments看不出来问题

**Comment by shenchenbing** [comment]
> callbacks = [ keras.callbacks.ModelCheckpoint( filepath="convnet_from_scratch.keras", save_best_only=True, monitor="val_loss") ] history = model.fit( train_dataset, epochs=30, validation_data=validation_dataset, callbacks=callbacks)这步模型训练报错，input arguments看不出来问题

具体是报了什么错？
我倒是没碰到报错的情况，就是自己跑出来的测试精度和书里面有一些差异。

**Comment by WalkCharlie** [comment]
改成不用GPU就不报错。发自我的 iPhone在 2024年7月24日，14:33，shenchenbing ***@***.***> 写道：﻿

callbacks = [ keras.callbacks.ModelCheckpoint( filepath="convnet_from_scratch.keras", save_best_only=True, monitor="val_loss") ] history = model.fit( train_dataset, epochs=30, validation_data=validation_dataset, callbacks=callbacks)这步模型训练报错，input arguments看不出来问题

具体是报了什么错？
我倒是没碰到报错的情况，就是自己跑出来的测试精度和书里面有一些差异。
